src/NServiceBus.Metrics.Tests/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -45,8 +39,8 @@ public class MetricsOptions
             "om version 3.0.0. Will be removed in version 3.0.0.", false)]
         public void EnableMetricTracing(System.TimeSpan interval) { }
         public void RegisterObservers(System.Action<NServiceBus.ProbeContext> register) { }
-        [System.ObsoleteAttribute("Not for public use. Will be treated as an error from version 2.0.0. Will be remov" +
-            "ed in version 3.0.0.", false)]
+        [System.ObsoleteAttribute("Not for public use. Will be treated as an error from version 3.0.0. Will be remov" +
+            "ed in version 4.0.0.", false)]
```

Comment:
I'm not sure what to do about this. It was only ever obsoleted to prevent customers from using it while we worked out the details of the monitoring stuff. It should eventually be made available. Can we just release minors with this obsolete attribute gone?

Reply:
@mikeminutillo the paln is to move this thing out of here. Once we have `NServiceBus.Metrics.ServiceControl` package it will be moved there.